tests/comp_options-textformat.js

```diff
@@ -7,21 +7,28 @@ import \"google/protobuf/descriptor.proto\";\
 message MyOptions {\
   string a = 1;\
   string b = 2;\
+  MyOptions c = 3;\
+  repeated string d = 4;\
+  repeated MyOptions e = 5;\
 }\
 extend google.protobuf.FieldOptions {\
   MyOptions my_options = 50000;\
 }\
 message Test {\
   string value = 1 [(my_options) = { a: \"foo\" b: \"bar\" }];\
-  string value2 = 2 [(my_options) = { a: \"foo\" b { c: \"bar\" } }];\
+  string value2 = 2 [(my_options) = { a: \"foo\" c { a: \"bar\" } }];\
   string value3 = 3 [(my_options) = { a: \"foo\", b: \"bar\" }];\
+  string value4 = 4 [(my_options) = { d: [\"foo\", \"bar\"] }];\
+  string value5 = 5 [(my_options) = { e: [ { d: [\"foo\", \"bar\"] } ] }];\
```

Comment:
This isn't valid protobuf. array notation [] is not valid here

Reply:
(Sorry, I seem to have missed the notification for this comment when it came in a month ago.)

I don't think that's true. My company's code base uses it. Also, I made sure that this protobuf message was accepted by protoc before submitting the pull request.

I'm fine with dropping the pull request though, as we've decided against dealing with protobufs in Javascript. It's up to you.